core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/arraypanes/ArrayLine.java

```diff
@@ -379,14 +375,40 @@ public int compareTo(final java.lang.Object other) {
     if (!(cell instanceof Comparable)) {
       return 0;
     }
-    sort = ((Comparable) cell).compareTo(tmp.getCellAt(sort));
+    sort = ((Comparable) cell).compareTo(other.getCellAt(sort));
     if (pane.getColumnToSort() < 0) {
       return -sort;
     } else {
       return sort;
     }
   }
 
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof ArrayLine)) {
+      return false;
+    }
+
+    final ArrayLine arrayLine = (ArrayLine) o;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "arrayLine" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)